### PR TITLE
Add domain reclamation UI.

### LINKED ISSUE #379: Re-claim subdomain
As a user I want to re-claim a subdomain I previously registered so I don't lose my subdomain if I re-install or reset my gateway

### PR DESCRIPTION
**Pull request by mrstegeman**
Fixes #379

### CODE REVIEW

**Comment by mrstegeman** [comment]
Diff is cleaner if you add `?w=1` to the URL.

File: src/controllers/settings_controller.js [hunk 1]
```diff
@@ -72,8 +72,23 @@ SettingsController.get('/experiments/:experimentName',
...
+SettingsController.post('/reclaim', async (request, response) => {
+  let subdomain = request.body.subdomain;
+
+  try {
+    await fetch(config.get('ssltunnel.registration_endpoint') +
+                '/reclaim?name=' + subdomain);
+    response.status(200).end();
+  } catch (e) {
+    console.error(e);
+    response.statusMessage = 'Error issuing certificate - ' + e;
```
Comment (hobinjk) [inline src/controllers/settings_controller.js]: This error message sounds out-of-place

**Comment by hobinjk** [comment]
@mrstegeman Under what conditions can a domain be reclaimed? I tried testing by registering a domain, verifying an email, then resetting and trying to register the same domain. The Gateway just said the domain was in use without giving me the option to send a reclamation email

**Comment by mrstegeman** [comment]
Fixed the error message.

Domains can be reclaimed if they have an email address associated with them, and the same email is provided when trying to reclaim. The required server changes are only active on the staging server right now, so if you'd like to try, just switch to mozilla-iot-staging.org.

**Comment by mrstegeman** [comment]
Good call, @hobinjk. I removed the call to /setemail when reclaiming.

**Pull request merged**